code/formatters/spdx.py

```diff
@@ -36,7 +38,12 @@ def create_base_spdx_document(name) -> Document:
     return newDocument
 
 def create_spdx_namespace(documentName: str) -> str:
-    return f'{SPDX_NAMESPACE_BASE}{documentName}-{uuid4()}'
+    namespace = f'{SPDX_NAMESPACE_BASE}{documentName}-{uuid4()}'
+    sha1_hash = sha1(namespace.encode('utf-8')).hexdigest()
+    sha512_hash = sha512(namespace.encode('utf-8')).hexdigest()
+    # return a dictionary containing both hash values
+    return {'sha1': sha1_hash, 'sha512': sha512_hash}
```

Comment:
I'm not quite sure I understand why this change was made. This section of code does not require any sort of SHA hashing. This change should be reverted before merging. Please reference the Namespace section of the SPDX spec here: https://spdx.dev/spdx-specification-21-web-version/#h.1gdfkutofa90